tasks/dashboard.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # tasks/dashboard.yml: Deployment of the Uchiwa dashboard
-  - include: "{{ ansible_distribution }}/dashboard.yml"
+  - include: "{{ role_path }}/tasks/{{ ansible_distribution }}/dashboard.yml"
```

Comment:
Include already looks in that directory automatically, why does it need to change here and the other places?